packages/rx_bloc_cli/mason_templates/bricks/feature_login/__brick__/test/feature_login/blocs/login_test.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:rx_bloc_test/rx_bloc_test.dart';
+import 'package:{{project_name}}/base/common_blocs/coordinator_bloc.dart';
+import 'package:{{project_name}}/base/models/errors/error_model.dart';
+import 'package:{{project_name}}/feature_login/blocs/login_bloc.dart';
+import 'package:{{project_name}}/feature_login/services/login_validator_service.dart';
+import 'package:{{project_name}}/lib_auth/services/user_account_service.dart';
+
+import 'login_test.mocks.dart';
+
+@GenerateMocks([
+  CoordinatorBlocType,
+  UserAccountService,
+  LoginValidatorService,
+])
+void main() {
+  late CoordinatorBlocType coordinatorBloc;
+  late UserAccountService userAccountService;
+  late LoginValidatorService validatorService;
+
+  LoginBloc loginBloc() => LoginBloc(
+        coordinatorBloc,
+        userAccountService,
+        validatorService,
+      );
+  setUp(() {
+    coordinatorBloc = MockCoordinatorBlocType();
+    userAccountService = MockUserAccountService();
+    validatorService = MockLoginValidatorService();
+  });
+
+  group('test login_bloc_dart state email', () {
+    rxBlocTest<LoginBlocType, String>('test login_bloc_dart state email',
+        build: () async {
+          return loginBloc();
+        },
+        act: (bloc) async {},
+        state: (bloc) => bloc.states.email,
+        expect: []);
```

Comment:
These tests seem to be missing... they're not `expect`ing anything

Reply:
I fixed this. Can you review again and resolve the conversation if you think it's fixed.